packages/toolkit/src/view/hub/hub-view/Body.tsx

```diff
@@ -105,30 +121,36 @@ export const Body = ({
                 <Icons.SearchSm className="my-auto h-4 w-4 stroke-semantic-fg-primary" />
               </Input.LeftIcon>
               <Input.Core
-                value={searchCode ?? ""}
+                value={searchValue ?? ""}
```

Comment:
@iamnamananand996 Let's make this state more clearer. Let call it searchInputValue

Reply:
Hi @EiffelFly, update all suggested changes